src/app/public/modules/split-view/split-view-drawer.component.ts

```diff
@@ -0,0 +1,207 @@
+  Observable,
+  Subject
+} from 'rxjs';
+
+import {
+  SkySplitViewMediaQueryService
+} from './split-view-media-query.service';
+
+import {
+  SkySplitViewService
+} from './split-view.service';
+
+let nextId = 0;
+
+@Component({
+  selector: 'sky-split-view-drawer',
+  templateUrl: 'split-view-drawer.component.html',
+  styleUrls: ['split-view-drawer.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [
+    SkySplitViewMediaQueryService,
+    { provide: SkyMediaQueryService, useExisting: SkySplitViewMediaQueryService }
+  ]
+})
+export class SkySplitViewDrawerComponent implements AfterViewInit, OnInit, OnDestroy {
+
+  @Input()
+  public ariaLabel: string;
+
+  public isMobile = false;
+
+  @Input()
+  public set width(value: number) {
+    if (value) {
+      this._width = Number(value);
+      this.updateBreakpoint();
+      this.splitViewService.updateDrawerWidth(this._width);
+      this.changeDetectorRef.markForCheck();
+    }
+  }
+
+  public get width(): number {
+    if (this.isMobile) {
+      return undefined;
+    }
+    if (this._width > this.widthMax) {
+      return this.widthMax;
+    } else if (this._width < this.widthMin) {
+      return this.widthMin;
+    } else {
+      return this._width || this.widthDefault;
+    }
+  }
+
+  public splitViewDrawerId: string = `sky-split-view-drawer-${++nextId}`;
+
+  // Max needs to start as something to allow input range to work.
+  // This value is updated as soon as the user takes action.
+  public widthMax = 9999;
+
+  public widthMin = 100;
+
+  public widthDefault = 320;
+
+  public widthTolerance = 100;
+
+  private isDragging = false;
+
+  private ngUnsubscribe = new Subject<void>();
+
+  private xCoord = 0;
+
+  private _width: number;
+
+  constructor(
+    private changeDetectorRef: ChangeDetectorRef,
+    private coreAdapterService: SkyCoreAdapterService,
+    private elementRef: ElementRef,
+    private splitViewMediaQueryService: SkySplitViewMediaQueryService,
+    private splitViewService: SkySplitViewService
+  ) {}
+
+  public ngOnInit(): void {
+    this.setMaxWidth();
+
+    this.splitViewService.isMobileStream
+      .takeUntil(this.ngUnsubscribe)
+      .subscribe((mobile: boolean) => {
+        this.isMobile = mobile;
+        this.changeDetectorRef.markForCheck();
+      });
+  }
+
+  public ngAfterViewInit(): void {
+    this.updateBreakpoint();
+  }
+
+  public ngOnDestroy(): void {
+    this.ngUnsubscribe.next();
+    this.ngUnsubscribe.complete();
+  }
+
+  public onResizeHandleMouseDown(event: MouseEvent): void {
+    event.preventDefault();
+    event.stopPropagation();
+
+    if (this.splitViewMediaQueryService.isWidthWithinBreakpiont(window.innerWidth,
+      SkyMediaBreakpoints.xs)) {
+        return;
+    }
+
+    this.setMaxWidth();
+    this.isDragging = true;
+    this.xCoord = event.clientX;
+
+    this.coreAdapterService.toggleIframePointerEvents(false);
+
+    Observable
+      .fromEvent(document, 'mousemove')
```

Comment:
Do these work on mobile? Or do we need touch events too?

Reply:
Nope, but good thought. I just followed the flyout resize pattern here. Mind if we file this as a separate issue? It would take some considerable effort to test and this is just an alpha release.

Reply:
Sure